app/javascript/dashboard/routes/dashboard/settings/sla/SlaForm.vue

```diff
@@ -5,6 +5,11 @@
         v-model="name"
         :class="{ error: $v.name.$error }"
         class="w-full"
+        :styles="{
```

Comment:
Can't we just use the Tailwind Classes with !? For eg: `!rounded-lg`

Reply:
This is not the default tailwind `style` attribute, WootInput supports this so that the styles can be applied to the input component which is a child element in the DOM tree. Applying the `class` directly will only put the class on the root element of the component which is the label

Reply:
If need be, we can port the components from the onboarding here. We can add that without making the PR too big, lmk!

Reply:
@pranavrajs let me know what do you think

Reply:
@pranavrajs PTAL